src/client/pages/Parkeren/Parkeren.tsx

```diff
@@ -54,24 +55,21 @@ export default function Parkeren() {
 
   const pageContentTop = (
     <>
-      <Paragraph className="ams-mb--sm">
-        Alle informatie over parkeren in Amsterdam vindt u op . Daar kunt u ook
-        terecht voor informatie over fietskelders, laadpalen voor elektrische
-        auto&apos;s en andere vragen die U heeft over parkeren of vervoer. Het
-        aanvragen of wijzigen van een parkeervergunning voor bewoners kan via
-        Mijn Parkeren. U moet hier wel opnieuw inloggen.
-      </Paragraph>
-      {isLoadingParkerenUrl && (
-        <LoadingContent barConfig={[['210px', '40px', '0']]} />
-      )}
-      {!isLoadingParkerenUrl && parkerenUrlSSO && (
-        <MaLink href={parkerenUrlSSO}>
-          <Button>
-            Log in op Mijn Parkeren
-            <Icon svg={ExternalLinkIcon} size={'level-5'} />
-          </Button>
-        </MaLink>
-      )}
+      <ErrorAlert severity="info" title="Let op">
+        Het inzien, aanvragen of wijzigen van een parkeervergunning voor
+        bewoners kan via Mijn Parkeren.
+        {isLoadingParkerenUrl && (
+          <LoadingContent barConfig={[['210px', '40px', '0']]} />
+        )}
+        {!isLoadingParkerenUrl && parkerenUrlSSO && (
+          <MaLink href={parkerenUrlSSO}>
+            <Button>
+              Log in op Mijn Parkeren
+              <Icon svg={ExternalLinkIcon} size={'level-5'} />
+            </Button>
+          </MaLink>
```

Comment:
Een <button/> is geen valide child element van. een <a/>. Er kan hier een button gebruikt worden.

```
<Button
  variant="primary"
>
  Button label
  <Icon
    size="level-5"
    svg={function ra(){}}
  />
</Button>
```

https://designsystem.amsterdam/?path=/docs/components-buttons-button--docs